c++/include/orc/Reader.hh

```diff
@@ -266,6 +286,12 @@ namespace orc {
      * Get desired timezone to return data of timestamp type
      */
     const std::string& getTimezoneName() const;
+
+    /**
+     * Get the <typeId, ArrayReadIntent> map that was supplied through
```

Comment:
Please use `TypeReadIntents` instead of `<typeId, ArrayReadIntent>`.

Reply:
Deleted in latest commit.